### PR TITLE
chore(ci): fix django snapshot timeouts

### PR DESCRIPTION
**Pull request by avara1986**
This PR improves and fixes issues in the Django tests with the following changes:
- Share the PATH and PYTHONPATH with the subprocess: When a script runs in a subprocess, there are times in the CI or locally when it's assigned a different path than expected. Even worse, we've seen scripts that worked for months suddenly stop working because of this. With this function, we always set the path to ensure consistent results both locally and across different CI environments
- Replace the `ddtrace-run` command with `python -m ddtrace.commands.ddtrace_run` to ensure that the local library is executed, rather than a potentially installed version
- Displays the stdout and stderr if there was an issue in the subprocess

## Checklist
- [x] PR author has checked that all the criteria below are met
- The PR description includes an overview of the change
- The PR description articulates the motivation for the change
- The change includes tests OR the PR description describes a testing strategy
- The PR description notes risks associated with the change, if any
- Newly-added code is easy to change
- The change follows the [library release note guidelines](https://ddtrace.readthedocs.io/en/stable/releasenotes.html)
- The change includes or references documentation updates if necessary
- Backport labels are set (if [applicable](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting))

## Reviewer Checklist
- [x] Reviewer has checked that all the criteria below are met 
- Title is accurate
- All changes are related to the pull request's stated goal
- Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes
- Testing strategy adequately addresses listed risks
- Newly-added code is easy to change
- Release note makes sense to a user of the library
- If necessary, author has acknowledged and discussed the performance implications of this PR as reported in the benchmarks PR comment
- Backport labels are set in a manner that is consistent with the [release branch maintenance policy](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting)

### CODE REVIEW

**Review by christophe-papazian — APPROVED** [review]

**Review by juanjux — APPROVED** [review]

File: tests/internal/test_module.py [hunk 1]
```diff
@@ -10,19 +10,11 @@
...
-
-
-def _build_env():
-    environ = dict(PATH="%s:%s" % (ROOT_PROJECT_DIR, ROOT_DIR), PYTHONPATH="%s:%s" % (ROOT_PROJECT_DIR, ROOT_DIR))
-    if os.environ.get("PATH"):
-        environ["PATH"] = "%s:%s" % (os.environ.get("PATH"), environ["PATH"])
-    if os.environ.get("PYTHONPATH"):
-        environ["PYTHONPATH"] = "%s:%s" % (os.environ.get("PYTHONPATH"), environ["PYTHONPATH"])
-    return environ
+FILE_PATH = os.path.dirname(os.path.abspath(__file__))
```
Comment (P403n1x87) [inline tests/internal/test_module.py:17]:
```suggestion
FILE_PATH = Path(__file__).resolve().parent
```

File: tests/utils.py [hunk 2]
```diff
@@ -47,6 +47,8 @@
     import importlib_metadata
 
 NO_CHILDREN = object()
+DDTRACE_PATH = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```
Comment (P403n1x87) [inline tests/utils.py:50]:
```suggestion
DDTRACE_PATH = Path(__file__).resolve().parents[-2]
```

**Review by taegyunkim — APPROVED** [review]

**Review by wconti27 — APPROVED** [review]

File: tests/utils.py [hunk 3]
```diff
@@ -1358,3 +1357,20 @@ def decorator(function_or_class):
         return _get_skipped_item(function_or_class, full_reason)
 
     return decorator
+
+
+def _build_env(env=None, file_path=FILE_PATH):
+    """When a script runs in a subprocess, there are times in the CI or locally when it's assigned a different
+    path than expected. Even worse, we've seen scripts that worked for months suddenly stop working because of this.
+    With this function, we always set the path to ensure consistent results both locally and across different
```
Comment (erikayasuda) [inline tests/utils.py:1365]: 🎉

**Review by erikayasuda — APPROVED** [review]
Thank you Alberto!! 🎉

**Pull request merged**